packages/calypso-build/typescript/js-package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig",
+
+	"compilerOptions": {
+		// Allow composed projects
```

Comment:
I don't think comments are allowed in `*.json` files?

Reply:
It's complicated. JSON (https://tools.ietf.org/html/rfc8259) does not allow comments, but other formats like JSON5 do (https://spec.json5.org/). Traditionally both files use `.json` extension, so looking at the extension is not enough. And TypeScript doesn't say which format the file should be in (https://www.typescriptlang.org/docs/handbook/tsconfig-json.html), but they do accept coments (https://github.com/microsoft/TypeScript/issues/4987).

That being said, even adding comments here works, I'm going to remove them because it confuses lots of tools and they don't really provide any value.